bin/compile

```diff
@@ -93,6 +93,11 @@ fi
 
 speak "Running stack"
 cd "$WORK_DIR"
+# Makefile support (if you need other build tools than Stack that you need to use in conjunction)
+if [ -f $BUILD_DIR/Makefile ]; then
+    speak "Makefile detected"
+    make -C $BUILD_DIR
+fi
```

Comment:
I will put in the quotes! As for the else-clause, I thought about it, but it's not a given that the stack build command with the exact flags "stack build --fast --copy-bins" will be run in the Makefile preceding it. I thought it was safer to run it outside the clause, because in the worst case scenario, Stack will just say "but I already did that just now!" and not build it twice.